api/src/main/resources/liquibase.xml

```diff
@@ -402,7 +402,7 @@ VALUES
 	<changeSet id="ugandaemr-10072020-1433" author="mmwanje">
 		<preConditions onFail="MARK_RAN">
 			<sqlCheck expectedResult="0">
-				SELECT COUNT(*) from role WHERE uuid =‘130106e6-04b4-11ea-8d71-362b9e155667’
+				SELECT COUNT(*) from role WHERE uuid ='130106e6-04b4-11ea-8d71-362b9e155667'
```

Comment:
Why are you using a select

Reply:
@slubwama what should he use in the pre-condition?

Reply:
Actually this is fine just go ahead and merge